lib/child_process.js

```diff
@@ -46,7 +46,7 @@ function stdioStringToArray(option) {
     case 'inherit':
       return [option, option, option, 'ipc'];
     default:
-      throw new TypeError('Incorrect value of stdio option: ' + option);
+      throw new TypeError(`Incorrect value of stdio option: ${option}`);
```

Comment:
There is another PR open that will be landing soon that changes this error message. It's likely safe to omit this change here and just wait for the other PR to land.